### ISSUE TITLE
Is this library long-lasting? 

### ISSUE THREAD

**Issue by archnim** [Vladar4/sdl2_nim#32]
Hello world. This may not be the right place to ask my question. If it's not, I'm sorry.
I prefer this version of sdl2 than the official repo's. But I ask myself : How can I be sure that it will still be maintained in several years. 
Please do you plan to keep maintaining this library?

**Comment by antwgames** [comment]
Hello there,

I'm just a fellow user, I'm not part of any Nim team or of this project. I'm not even a Nim 'expert', but judging from my fiddling with Nim and Nim's tooling and documentation I honestly don't think you need to worry about this too much - or at all:

1. I consider SDL2 as mature and this project is just a wrapper.
2. As you go along, I am sure you will be able to either maintain this wrapper by yourself if need be - or even to recreate it from scratch.
3. There's actually a book about Nim: "Nim in Action". The book author has this website: https://book.picheta.me/ - and there the author offers two sample chapters, including chapter 8. This chapter 8 actually teaches you how to create a wrapper library in Nim - not only that, the book even uses SDL 2 as the example. The free chapters come as PDF file(s). If you like the sample content, the author sure might appreciate if you decide to purchase a copy (I am not affiliated). Manning runs some Christmas sales. They might be over very soon, but maybe they'll have some 50% ish discount for this book. One or two days ago they offered 50% off of everything - that's over, but maybe there will be another chance soon, I don't know. Sorry if this sounds like an advertisement, it really isn't. I am not the author and I am not affiliated with Manning by any means.

To summarize: I do not think you will depend on this library being maintained years in advance. Sometimes you have to do things that seem far more difficult in your projects - and far more daunting. If you like the authors approach for this wrapper library, I really don't think you'll have to worry about things years down the road. Taking advantage of C libraries (like SDL2) probably is among the strengths of the Nim language in my opinion. Those things are not an afterthought at all given the nature and the history of the language.


---------

For the record: I don't want to downplay Vladar4's work by any means. Quite the opposite, please accept my thanks for sharing this with the world.

**Comment by ftsf** [comment]
I use this library a lot as it's a big dependency of Nico and if @Vladar4 no longer wishes to maintain it in the future I'd be happy to take over, but it seems like it's pretty well maintained, considering SDL2 isn't doing many major changes these days.

**Comment by archnim** [comment]
Antw games, thank you very much. I would like us the become pals. You can write to me here: https://twitter.com/archnim.
Thanks greatly ftsf. ❣️

**Comment by Vladar4** [comment]
> Please do you plan to keep maintaining this library?

Yes.

**Comment by archnim** [comment]
Ok Vlad4. I admire those who work hard to make Nim greater. I would like to know your opinion about my personal project (a very crazy idea). Please would you like to advice me about that on my Twitter account?
